blacksauce/src/styles/_utilityClasses.scss

```diff
@@ -0,0 +1,72 @@
+.u-brandOrange {
+    color: #F28705;
+}
+
+.u-fontWeightBold {
+    font-weight: 700;
+}
+
+.u-fontWeightReg {
+    font-weight: 400;
+}
+
+.u-black {
+    background-color: black;
+}
+
+.u-white {
+    color: white;
+}
+
+.u-transparentBlack {
+    background-color: rgba(0, 0, 0, .5);
+    border-radius: 75px;
+    padding: 5%;
+    position: relative;
+}
+
+.u-marginNone {
+    margin: 0;
+}
+
+.u-block {
+    display: block;
+}
+
+.u-button-reset {
+    background: transparent;
+    border: 0;
+    cursor: pointer;
+    font-family: inherit;
+    font-size: inherit;
+    font-weight: inherit;
+    padding: 0;
+    text-align: inherit;
+}
+
+.u-list-reset {
+    list-style: none;
+    padding-left: 0;
+}
+
+.footerPadding {
+    padding: 5%;
+}
+
+
+a {
```

Comment:
All the `a` styles should go either in a `_base.scss` file (that only has styles for bare elements, no classnames) or your existing `_text.scss` partial.